Dockerfile

```diff
@@ -1,12 +1,14 @@
 FROM jupyter/datascience-notebook
 
-MAINTAINER Pierre Bellec <pierre.bellec@gmail.com>
+LABEL maintainer="Pierre Bellec <pierre.bellec@gmail.com>"
 
 USER jovyan
 
-RUN pip install nilearn
+# Copying the repository inside the container
+COPY . /home/jovyan 
 
-# Get the files of the project
-RUN wget https://github.com/brainhack101/introML/archive/0.1.zip
-RUN unzip 0.1.zip
-RUN rm 0.1.zip
+# Instaling the Kamalaker's main fecther
+RUN pip install --user git+https://github.com/KamalakerDadi/nilearn.git@16f2df26401e5b6a16ff134f4da71ff920e3ac40
```

Comment:
I think for Jupyter Book (and for general flexibility) we should just install a `requirements.txt`. Could we update that, here ?

Reply:
Yep you are right,
In the future we should do the same for the data instead of `fetch_data.sh`

Reply:
:+1: thanks !